visualization/app/codeCharta/ui/viewCube/viewCube.mouseEvents.service.spec.ts

```diff
@@ -4,6 +4,7 @@ import { ViewCubeMouseEventsService } from "./viewCube.mouseEvents.service"
 // eslint-disable-next-line no-duplicate-imports
 import * as Three from "three"
 import oc from "three-orbit-controls"
+import type { OrbitControls } from "three/examples/jsm/controls/OrbitControls"
```

Comment:
I totally agree. I am not sure, but I guess it is hard coupled to our also very deprecated Three.js? What do you think about an own ticket for updating Three.js and its companions?

Reply:
That ticket already exists as #2492 👀 
I updated it with the need to move away from the orbit controls

Reply:
@shaman-apprentice three.js is actively developed and not deprecated, is it?

@MW-Friedrich I might be wrong but it seems like there's an example file in the three.js repository that would reflect a current OrbitControls alternative and we could just use that.

Reply:
Yes, there is! Orbit or Arcball controls are quite common. I have updated the issue again accordingly 😄

Reply:
Three.js is actively development. Current version is 0.148.0 and we are on 0.126.1. To be honest I have no idea how much work it will be to upgrade to latest version.